generic/tclcurl.h

```diff
@@ -239,11 +243,11 @@ CONST static char *configTable[] = {
     "-share",             "-port",               "-tcpnodelay",
     "-autoreferer",       "-sourcehost",         "-sourceuserpwd",
     "-sourcepath",        "-sourceport",         "-pasvhost",
-    "-sourceprequote",    "-sourcepostquote",    "-ftpsslauth",
-    "-sourceurl",         "-sourcequote",        "-ftpaccount",
-    "-ignorecontentlength",                      "-cookielist",
-    "-ftpskippasvip",     "-ftpfilemethod",      "-localport",
-    "-localportrange",
+    "-sourceprequote",    "-sourcepostquote",
+    "-ftpsslauth",        "-sourceurl",          "-sourcequote",
+    "-ftpaccount",        "-ignorecontentlength",
+    "-cookielist",        "-ftpskippasvip",
+    "-ftpfilemethod",     "-localport",          "-localportrange",
```

Comment:
Was anything actually changed here?

Reply:
No, it's easy to verify that the 2 option lists are just reorganized